kubeflow/hack/create_context.sh

```diff
@@ -6,24 +6,17 @@
 # PROJECT=<PROJECT> REGION=<region> NAME=<NAME>
 #
 # TODO(jlewi): Support zonal clusters as well
-set -x 
+set -x
 
-# Default namespace to kubeflow
-NAMESPACE=${NAMESPACE:-kubeflow}
-echo Checking if context ${NAME} exists 
-
-kubectl config use-context ${NAME}
-
-RESULT=$?
-
-if [ ${RESULT} -eq 0 ]; then
-echo kubeconfig context ${NAME} already exists
-exit 0
-fi
+# Delete kubeconfig context if exist already
```

Comment:
What do you think?
nit:
```
# Always delete the existing kubeconfig, because it may be an outdated
# context to a deleted cluster with the same name.
```

Reply:
Just omitting `Always` because it will make sense automatically every time when updating or installing.
Is that okay?